### ISSUE TITLE
[Feature request] Planned task completion date field

### ISSUE THREAD

**Issue by lectrician1** [nextcloud/tasks#2107]
I use tasks to track my homework and often times I want to plan when I will complete my homework before it is due. 

Having a field that allows me to select a day or period of time when I plan to get a task completed would be nice solution for this.

**Comment by raimund-schluessler** [comment]
We follow the CalDAV specification and this does not support such a date. It only allows start, due and completed dates for VTODOs, see https://www.rfc-editor.org/rfc/rfc5545#section-3.8.2.

**Comment by lectrician1** [comment]
Does this app have to follow the specification? Such a change would not impact the implementation of the specification at all. It seems silly that useful extensible features like this are not possible.

**Comment by lectrician1** [comment]
Any response?

**Comment by raimund-schluessler** [comment]
Yes, we follow the CalDAV standard here. Of course we could still implement custom features. But to be honest, I don't see a use case for a specific planned completion date field. In my opinion, this is exactly what the start date field is for.

**Comment by lectrician1** [comment]
Yes, using the start date is a good option. Thank you.
